NEP/donations.yml

```diff
@@ -0,0 +1,41 @@
+- ticker: NEP
+  recipient: 
+    name: Street Child
+    address: Street Child - 206-208 Stewarts Road, Larkhall, London, SW8 4UB, UK
+    tel: 020 7614 7696
+    contact: info@street-child.org
+  ada: 105.00 # <- These must be numbers. Can be decimals though. See below
+  usd: 110.00 # Only usd currency accepted, check on xe.com for exchange rate.
+  timestamp: 1624966028 
+  files:
+    - NEP/2020-06-29-SC.jpeg
```

Comment:
Hey there, thanks for submitting your valuable contributions to these Charities!
The screenshots of the donation receipts from your mailbox include the necessary information needed to establish a base layer of trust in your claim. What the MDP is trying to achieve here is to strengthen the trust in SPO donation claims, so any information (bank receipts, paypal, blockchain transactions) that you can produce to strengthen that trust is greatly appreciated and makes it easier for the MDP memebers that process PRs to approve your claim.
Please note that files like `2020-06-29-SC.jpeg`, `2020-06-29-DD.jpeg` or `2020-06-29-TA-1.jpeg` do not really provide any sort of information about your donation since they reference neither a sender nor an amount.  So in the future you don't need to upload these.